sdk-core/src/main/java/com/reach5/identity/sdk/core/JavaReachFive.kt

```diff
@@ -22,8 +22,12 @@ class JavaReachFive(activity: Activity, sdkConfig: SdkConfig, providersCreators:
         return reach5.getProviders()
     }
 
+    fun loginWithNativeProvider(name: String, scope: Collection<String>, origin: String, activity: Activity) {
+        return reach5.loginWithProvider(name, scope, origin, activity)
+    }
+
     fun loginWithNativeProvider(name: String, origin: String, activity: Activity) {
-        return reach5.loginWithProvider(name, origin, activity)
+        return reach5.loginWithProvider(name, emptySet(), origin, activity)
```

Comment:
Maybe you pass other arguments as name-value to prevent the use of this emptySet ?

Reply:
Sorry, I didn't understand.

Reply:
I don't know the actual keyword but I imagine sth like
```reach5.loginWithProvider(name = name, origin = origin, activity = activity)```

Reply:
But you can't do that in Java. This is `JavaReachFive` API.